packages/flutter/lib/src/foundation/consolidate_response.dart

```diff
@@ -44,7 +44,7 @@ typedef BytesReceivedCallback = void Function(int cumulative, int total);
 /// bytes from this method (assuming the response is sending compressed bytes),
 /// set both [HttpClient.autoUncompress] to false and the `autoUncompress`
 /// parameter to false.
-Future<TransferableTypedData> consolidateHttpClientResponseBytes(
+Future<TransferableTypedData> consolidateHttpClientResponseBytesTransferable(
```

Comment:
This is going to be the method that remains after we remove the deprecated one, so I'd rather we go with a shorter method name.

How about `getHttpClientResponseBytes()`?

Reply:
Done